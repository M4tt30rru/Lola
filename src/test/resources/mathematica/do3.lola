##Find do[##Any(_e),{##Any(_i),##Any(_list)}];
 ##replace for(##(_i):##(_list)) {##(_e);}

do[f(o),{MyObject o,myOs}];
