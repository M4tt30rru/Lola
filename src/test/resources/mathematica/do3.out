for(MyObject o:myOs) {f(o);}